shield-config.js

```diff
@@ -1,16 +1,16 @@
 /* global require, exports:false */
 
 const self = require('sdk/self');
-const SHIELD_PREF = 'extensions.tabcentertest1@mozilla.com.shield';
 const {get, set} = require('sdk/preferences/service');
 let study_duration = 14; // in days
 
 const studyConfig = {
   name: self.addonId,
   duration: study_duration,
   variations: {
-    'control': () => {set(SHIELD_PREF, 'control')},
-    'enabled': () => {set(SHIELD_PREF, 'enabled')}
+    'control': () => {},
```

Comment:
Do we need an empty function here, or could we make it `null`, or `true` instead?  Or even just all the same empty function, so that we don't have three of them hanging around, saving bindings to stuff…